cmd/gemini/status.go

```diff
@@ -97,6 +97,7 @@ func sampleStatus(ctx context.Context, c chan Status, sp *spinningFeedback, logg
 			testRes.Merge(res)
 			sp.Set(" Running Gemini... %v", testRes)
 			if testRes.ReadErrors > 0 || testRes.WriteErrors > 0 {
+				fmt.Printf("%#v", testRes.Errors)
```

Comment:
Let's improve this error message a bit to make it less cryptic for users with something like:

```go
fmt.Printf("Errors detected: %#v", testRes.Errors)
```